### ISSUE TITLE
ZBL empirical correction is broken

### ISSUE THREAD

**Issue by lkkmpn** [apax-hub/apax#373]
When a ZBL empirical correction is included, model initialization crashes with the following error
```
TypeError: Gradient only defined for scalar-output functions. Output had shape: (1,).
```

Example model configuration:
```yaml
model:
  empirical_corrections:
  - name: zbl
    r_max: 1.5
  # ...
```

The `exponential` empirical correction works fine.

**Comment by M-R-Schaefer** [comment]
thanks, I'll look into it

**Comment by M-R-Schaefer** [comment]
fixed by #362

**Comment by M-R-Schaefer** [comment]
should be fixed now on main
